local-interchaintest/src/tests/remote_chain_splitter/remote_chain_splitter.rs

```diff
@@ -0,0 +1,307 @@
+    let neutron = test_ctx.get_chain(NEUTRON_CHAIN_NAME);
+
+    reqwest::blocking::Client::default()
+        .post(&neutron.rb.api)
+        .json(&serde_json::json!({ "chain_id": NEUTRON_CHAIN_ID, "action": "start-relayer"}))
+        .send()
+        .unwrap();
+}
+
+fn stop_relayer(test_ctx: &mut TestContext) {
+    let neutron = test_ctx.get_chain(NEUTRON_CHAIN_NAME);
+
+    reqwest::blocking::Client::default()
+        .post(&neutron.rb.api)
+        .json(&serde_json::json!({ "chain_id": NEUTRON_CHAIN_ID, "action": "stop-relayer"}))
+        .send()
+        .unwrap();
+}
+
+fn upload_contracts(test_ctx: &mut TestContext) -> Result<(), LocalError> {
+    let mut uploader = test_ctx.build_tx_upload_contracts();
+
+    uploader
+        .send_with_local_cache(VALENCE_PATH, NEUTRON_CHAIN_NAME, LOCAL_CODE_ID_CACHE_PATH)
+        .unwrap();
+
+    uploader
+        .send_with_local_cache(ASTROPORT_PATH, NEUTRON_CHAIN_NAME, LOCAL_CODE_ID_CACHE_PATH)
+        .unwrap();
+
+    uploader.send().unwrap();
+
+    Ok(())
+}
+
+fn make_remote_chain_splitter(test_ctx: &mut TestContext) -> Result<ContractInfo, LocalError> {
+    let atom_denom = test_ctx.get_native_denom().src(GAIA_CHAIN_NAME).get();
+    let uatom_contribution_amount: u128 = 5_000_000_000;
+
+    let split_config: Vec<(String, SplitConfig)> = vec![(
+        atom_denom.clone(),
+        SplitConfig {
+            receivers: BTreeMap::from_iter(vec![
+                (ACC1_ADDRESS_NEUTRON.to_owned(), Decimal::percent(50)),
+                (ACC2_ADDRESS_NEUTRON.to_owned(), Decimal::percent(50)),
+            ]),
+        },
+    )];
+
+    let mut remote_chain_splitter = test_ctx
+        .get_contract("valence_remote_chain_splitter")
+        .unwrap();
+
+    remote_chain_splitter
+        .instantiate(
+            DEFAULT_KEY,
+            serde_json::to_string(&valence_remote_chain_splitter::msg::InstantiateMsg {
+                op_mode_cfg: ContractOperationModeConfig::Permissionless,
+                remote_chain_channel_id: test_ctx
+                    .get_transfer_channels()
+                    .src(NEUTRON_CHAIN_NAME)
+                    .dest(GAIA_CHAIN_NAME)
+                    .get(),
+                remote_chain_connection_id: test_ctx
+                    .get_connections()
+                    .src(NEUTRON_CHAIN_NAME)
+                    .dest(GAIA_CHAIN_NAME)
+                    .get(),
+                denom: atom_denom.clone(),
+                amount: Uint128::from(uatom_contribution_amount),
+                splits: BTreeMap::from_iter(split_config),
+                ica_timeout: Uint64::new(DEFAULT_TIMEOUT),
+                ibc_transfer_timeout: Uint64::new(DEFAULT_TIMEOUT),
+                fallback_address: None,
+            })
+            .unwrap()
+            .as_str(),
+            "valence_remote_chain_splitter",
+            None,
+            "",
+        )
+        .unwrap();
+
+    Ok(ContractInfo {
```

Comment:
In line with the first comment, here you can just return the Contract object you get from instantiate because you don't need the code_id for your tests.